lib/index.js

```diff
@@ -144,11 +146,15 @@ const readmeInspector = {
    * @memberOf module:readme-inspector
    * @param {string} url - The URL or slug of a repository.
    *
-   * @returns {ScoreData}
+   * @returns {AppraisalData}
    * @see https://github.com/clayallsopp/readme-score-api/#readme
    */
 
-  'getReadmeScore': ReadmeScore.for
+  async getReadmeScore (url) {
+    const readmeScore = new ReadmeScore()
+    const appraisal = await readmeScore.for(url)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "appraisal". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1488)

Reply:
### Prefer "Disallows unnecessary return await (no-return-await)"

> Inside an async function, return await is useless. Since the return value of an async function is always wrapped in Promise.resolve, return await doesn’t actually do anything except add extra time before the overarching Promise resolves or rejects. This pattern is almost certainly due to programmer ignorance of the return semantics of async functions.
>
> _no-return-await - Rules._ (2018). _ESLint - Pluggable JavaScript linter._ Retrieved 18 April 2018, from <https://eslint.org/docs/rules/no-return-await>